src/components/01-type/links.njk

```diff
@@ -1,9 +1,9 @@
-<p><a href="javascript:void(0);">This</a> is a text link on a light background.</p>
+<p><a class="usa-link" href="javascript:void(0);">This</a> is a text link on a light background.</p>
 
-<p><a class="usa-color-text-visited" href="javascript:void(0);">This</a> is a visited link.</p>
+<p><a class="usa-link usa-color-text-visited" href="javascript:void(0);">This</a> is a visited link.</p>
 
-<p>This is a link that goes to an <a class="usa-external_link" href="https://media.giphy.com/media/8sgNa77Dvj7tC/giphy.gif">external website</a>.</p>
+<p>This is a link that goes to an <a class="usa-link usa-external_link" href="https://media.giphy.com/media/8sgNa77Dvj7tC/giphy.gif">external website</a>.</p>
 
 <div class="usa-background-dark">
-  <p><a href="javascript:void(0);">This</a> is a text link on a dark background.</p>
+  <p><a class="usa-link" href="javascript:void(0);">This</a> is a text link on a dark background.</p>
 </div>
```

Comment:
Is a prose scope example represented somewhere on the site/documentation?

Reply:
Looks like we do use it on [Typesetting](https://v2.designsystem.digital.gov/components/typography/), though the documentation needs updating.

Reply:
Updating prose scope docs is next on my list